src/native/fs/fs_napi.cpp

```diff
@@ -991,6 +995,38 @@ struct FileSetxattr : public FSWrapWorker<FileWrap>
     }
 };
 
+struct linkFileAt : public FSWrapWorker<FileWrap>
+{
+    std::string _path;
+    std::string _filepath;
+    int _dirfd;
+    int _rc_res;
+
+
+    linkFileAt(const Napi::CallbackInfo& info)
+        : FSWrapWorker<FileWrap>(info)
+    {
+        _path = info[1].As<Napi::String>();
+        _filepath = info[2].As<Napi::String>();
```

Comment:
What is the difference between these? _path is the string provided to open, so for tempfile isn't that the same as the target path? or is there a case where I need to open with one path, but link it to another path?

Reply:
_path was the directory name but we don't need it here, open uses the directory since we don't have a name, but link at gives that temp file the name.

Reply:
I see, but is there a reason not to send the target file name on open temp file ('wt')?
Is open expecting a dir name there?

Reply:
yes, it must be the directory name

Reply:
ok, so is the _path argument still needed then? you can shift it to be arg number 1,
and call it like this - `await target_file.linkfileat(fs_account_config, file_path);`